package.json

```diff
@@ -6,32 +6,32 @@
     "deployer",
     "nscale"
   ],
-  "version": "0.17.9",
+  "version": "1.0.0",
   "license": "Artistic-2.0",
   "contributors": [
     "Peter Elger <elger.peter@gmail.com> (http://peterelger.com/)",
     "Matteo Collina <hello@matteocollina.com>",
     "Luca Lanziani <luca@lanziani.com>"
   ],
   "engines": {
-    "node": "*"
+    "node": ">=0.10 <=0.12"
   },
   "dependencies": {
-    "async": "^0.9.0",
+    "async": "~0.9.2",
```

Comment:
The tildes will allow minor updates to be pulled in.

I'm not sure if that's intentional or do we want to just keep it to patches?

Reply:
I believe tilde (~) only allows patches to be pulled in and carot (^) allows minor updates unless I have that backwards.

Reply:
Yup, you're correct. My bad >_<